src/js/store/orgs/actions.ts

```diff
@@ -348,3 +353,37 @@ export const commitFailed = ({
     payload: model,
   });
 };
+
+export const refreshOrg = (org: Org): ThunkResult => async (dispatch) => {
+  const url = window.api_urls.scratch_org_refresh(org.id);
+  dispatch({
+    type: 'REFRESH_ORG_STARTED',
+    payload: { org, url },
+  });
+  try {
+    const response = await apiFetch({
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      url: addUrlParams(url),
+      dispatch,
+      opts: {
+        method: 'POST',
+        body: JSON.stringify(org),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    });
+    if (!response) {
+      return dispatch({
+        type: 'REFRESH_ORG_FAILED',
+        payload: { org, url, response },
+      });
+    }
```

Comment:
I don't think we really care about the `response` for this request at all, so this failure case should be handled in the `catch` block below.